src/pages/index.tsx

```diff
@@ -90,100 +91,22 @@ const HomePage: React.FC<Props> = (props: Props) => {
       createdAt: new Date(bubble.createdAt)
     })))
   }, []);
-  
-  const postBubble = async (e, userInfo) => {
-    e.preventDefault();
 
-    const title = e.target.title.value;
-    const description = e.target.description.value;
-    const content = e.target.content.value;
-    const author = userInfo
-  
-    try {
-      await api.post('/bubbles', {
-        title,
-        description,
-        content,
-        author,
-      });
-      toast.success('Bubble successfully registered!', {
-        autoClose: 2500,
-        pauseOnHover: false,
-        pauseOnFocusLoss: false,
-      });
-      Router.push('/');
-      setIsNewBubbleModalVisible(false);
-
-    } catch {
-      toast.error('Registration error! Try again', {
-        autoClose: 2500,
-        pauseOnHover: false,
-        pauseOnFocusLoss: false,
-      });
-      Router.reload();
-    };
+  const postBubble = (e, bubblInfo, userInfo) => {
+    postBubbles(e, bubblInfo, userInfo);
+    setIsNewBubbleModalVisible(false);
   };
 
-  const postComment = async (e, userComment, userInfo) => {
-    e.preventDefault();
-
-    const content = userComment;
-    const author = userInfo;
-    const bubbleId = oppenedBubbleId;
-  
-    try {
-      await api.post('/comments', {
-        content,
-        author,
-        bubbleId,
-      });
-      toast.success('Comment registrated!', {
-        autoClose: 2500,
-        pauseOnHover: false,
-        pauseOnFocusLoss: false,
-      });
-      Router.reload();
-    
-    } catch {
-      toast.error('Registration error! Try again', {
-        autoClose: 2500,
-        pauseOnHover: false,
-        pauseOnFocusLoss: false,
-      });
-      Router.reload();
-    };
+  const postComment = (e, newComment, userInfo) => {
+    postComments(e, newComment, userInfo, oppenedBubbleId);
   };
 
-  const postLabel = async (e, newLabel) => {
-    e.preventDefault();
+  const postLabel = (e, newLabel) => {
+    postLabels(e, newLabel, oppenedBubbleId);
+  };
 
-    const name = newLabel.name;
-    const description = newLabel.description;
-    const color = newLabel.color;
-    const bubbleId = oppenedBubbleId;
-  
-    try {
-      await api.post('/labels', {
-        name,
-        description,
-        color,
-        bubbleId,
-      });
-      toast.success('Label registered!', {
-        autoClose: 2500,
-        pauseOnHover: false,
-        pauseOnFocusLoss: false,
-      })
-      Router.reload();
-
-    } catch {
-      toast.error('Registration error! Try again', {
-        autoClose: 2500,
-        pauseOnFocusLoss: false,
-        pauseOnHover: false,
-      })
-      Router.reload();
-    };
+  const alteredLabel = (e, id, selectedLabel) => {
+    alteredLabels(e, id , selectedLabel, oppenedBubbleId);
```

Comment:
it would be better if we e.preventDefault() on onSubmit={e => {e.preventDefault(); callback()} so we dont have to pass it around